docker-compose.yml

```diff
@@ -39,11 +19,9 @@ services:
     depends_on:
       - mongo-init-replica
     environment:
-      METEOR_DISABLE_OPTIMISTIC_CACHING: 1
       MONGO_URL: "mongodb://mongo:27017/reaction"
       MONGO_OPLOG_URL: "mongodb://mongo:27017/local"
       ROOT_URL: "http://localhost:3000"
-      BABEL_DISABLE_CACHE: 1 # This is needed to pick up changes to .graphql files. See https://www.npmjs.com/package/babel-plugin-inline-import#caveats
```

Comment:
It was for `babel-plugin-inline-import` package, which is now removed in favor of `orionsoft:graphql-compiler` Meteor package because that pkg properly reloads `.graphql` files and resolves the "...is defined in resolvers but not in schema" errors. `babel-plugin-inline-import` is still used when running Jest or the pure Node app.